added release notes up to 10.16.223.0 (#126)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/platform-10-17-0.md

<td>10.16.8.0</td>
</tr>

<tr>
<td>REST API</td>

> <td>The performance of the Alarm API has been improved for requests in which the resolved query parameter is used.</td>

Reviewer: ```suggestion
<td>The performance of the Alarm API has been improved for requests which use the resolved query parameter.</td>
```
"use" or "contain" above
